Sources/SimpleSalesforceSDK/Models/URLRequestBuilder.swift

```diff
@@ -0,0 +1,48 @@
+import Foundation
+
+public struct URLRequestConfig {
```

Comment:
Don't think this should have `URL` in the name...

```suggestion
public struct RequestConfig {
```